src/ontology/maxo-edit.owl

```diff
@@ -13002,6 +13022,170 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <h
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020018> (Flexion MRI)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/MAXO_0020018> "A type of MRI of the cervical spine designed to assess cervial myelopathy. Static views are taken with the neck in the neutral position and the mandible nearly perpendicular to the horizontal plane. To obtain the flexion/extension views, patients are instructed to perform the maximally tolerated flexion and extension actions of the neck in a non–weight-bearing position with supporting sponges placed under the head and shoulders.")
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020018> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020018> "2025-06-02T05:09:11Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020018> "PMID:35636662")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0020018> "Flexion MRI")
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0020018> <http://purl.obolibrary.org/obo/MAXO_0000424>)
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020019> (intradermal histamine injection)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/MAXO_0020019> "A diagnostic procedure whereby histamine is injected into a local of the skin. This causes wheal and flare when they are injected intradermally into normal subjects and may therefore play a role in the formation of urticaria.")
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020019> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020019> "2025-06-02T05:15:47Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020019> "This response may be absent in individuals with certin types of hereditary sensory and autonomic neuropathy.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/MAXO_0020019> "intradermal histamine injection")
+SubClassOf(<http://purl.obolibrary.org/obo/MAXO_0020019> <http://purl.obolibrary.org/obo/MAXO_0001404>)
+
+# Class: <http://purl.obolibrary.org/obo/MAXO_0020020> (D-xylose urine test)
+
+AnnotationAssertion(dce:creator <http://purl.obolibrary.org/obo/MAXO_0020020> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dce:date <http://purl.obolibrary.org/obo/MAXO_0020020> "2025-06-02T05:25:23Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020020> "PMID:10478874")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/MAXO_0020020> "The standard 25-g D-xylose test in adults, based on analysis of 5-h urine collection and a 1-h serum sample, discriminates between normal subjects and patients with proximal small intestinal malabsorption.")
```

Comment:
Probably wrong property, should be IAO:0000115.

Reply:
don't understand